### PR TITLE
Add menu

### PR DESCRIPTION
**Pull request by tiffanyrichards**
Added a menu with two options (1) add items or (2) exit the program. Takes user input and displays an exit message if user chooses the exit option.

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	cout << "1. Add item\n";
+	cout << "2. Exit\n";
+	cout << "Enter choice: ";
+
+	cin >> choice;
+	if (choice == 2) {
+	cout << "Goodbye!";
+	return 0;
+	}
+return 0;
```
Comment (JoshBranda) [inline menu.txt]: Formatting is off here.  Line 18 should be tabbed over.

File: menu.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+	int choice = 0;
+
```
Comment (JoshBranda) [inline menu.txt]: For a menu, you should always use a while loop.  So you need to encapsulate this dialogue / options in a while loop, and you can get rid of the if statement

File: assignment1.cpp [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+	int choice = 0;
+	
+	cout << "Main menu:" << endl;
```
Comment (JoshBranda) [inline assignment1.cpp]: Menus should always be inside of a while loop.  Use that instead, and you can remove the if statement at the bottom.

File: assignment1.cpp [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+	cout << "1. Add item\n";
+	cout << "2. Exit\n";
+	cout << "Enter choice: ";
+
+	cin >> choice;
+	if (choice == 2) {
+	cout << "Goodbye!";
+	return 0;
+	}
+return 0;
```
Comment (JoshBranda) [inline assignment1.cpp]: Bad formatting.  This should be tabbed over

File: menu.txt (hunk 1 above)
Comment (tiffanyrichards) [reply menu.txt]: Fixed

File: menu.txt (hunk 2 above)
Comment (tiffanyrichards) [reply menu.txt]: Fixed

**Review by JoshBranda — APPROVED** [review]

**Pull request merged**